cmd/validate.go

```diff
@@ -45,10 +45,9 @@ this command unless --online flag is used.
 		if err != nil {
 			return err
 		}
-
-		rawState, err := file.Get(targetContent, file.RenderConfig{
+		rawState, err := file.Get(cmd.Context(), targetContent, file.RenderConfig{
 			CurrentState: dummyEmptyState,
-		}, dump.Config{})
+		}, dump.Config{}, nil)
```

Comment:
Question: Why not populate defaults for validate when online flag is in use?

Reply:
Good catch, I started this PR before the online `validate` feature was merged and I forgot to add it back. I'll do it!

Reply:
Added